spec/support/travel_advice_publisher_navigation_helpers.rb

```diff
@@ -0,0 +1,79 @@
+module TravelAdvicePublisherNavigationHelpers
+  def visit_travel_advice_homepage
+    travel_advice_publisher_url("/admin")
+  end
+
+  def visit_draft_origin_url(path)
+    visit(Plek.find("draft-origin") + path)
+  end
+
+  def visit_published_url(path)
+    visit(Plek.find("www") + path)
+  end
+
+  def travel_advice_publisher_url(path)
+    visit(Plek.find("travel-advice-publisher") + path)
+  end
+
+  def create_new_edition_draft(title, summary, part_title, part_body)
+    visit_travel_advice_homepage
+    click_link("Argentina")
```

Comment:
Do you think it would be possible to make the test suite less dependent on a specific country (i.e. store this as some sort of variable/constant), allowing us to run the entire suite more than once on different countries if we wanted?